### ISSUE TITLE
Webinject enhancements

### ISSUE THREAD

**Issue by robertoschwald** [HeinleinSupport/check_mk_extensions#49]
I developed some enhancements for Webinject checks based on your extension.

https://github.com/robertoschwald/check_mk_plugins/tree/master/webinject/src

Enhancements:

- Packaged check_webinject Nagios check with the package (Version 1.94)
- Reusable test files, as you can specify the config file name and the test file name. With this, you can use a common test file for many hosts, but webinject config files per host.
- Configure username / password as needed by your webinject tests.

Feel free to move it into your plugin. 
Just leave a note when you've done, so I can remove it from my repo. Thanks.

**Comment by gurubert** [comment]
check_webinject is already part of the official CheckMK package.

For the rest please create a pull request.

**Comment by robertoschwald** [comment]
Which Check_MK version includes check_webinject? I searched all files in the 1.5.0p15 docker image, but didn't find it.

**Comment by gurubert** [comment]
```
OMD[heinlein]:~$ ./lib/nagios/plugins/check_webinject --version
WebInject version 1.88
OMD[heinlein]:~$ cmk --version
This is Check_MK version 1.5.0p16 CEE
```

**Comment by robertoschwald** [comment]
Ah, ok, thanks. One version newer..

**Comment by robertoschwald** [comment]
Please note that 1.5.0p15 does include check_webinject, but does not work in the Docker Container, as the Check_MK Docker image is missing Error.pm. The version I provide for now conditionally loads the Error module. Need to check if it is still the case with the newer versions.

OMD[demo]:~$ lib/nagios/plugins/check_webinject --version
Can't locate Error.pm in @INC (you may need to install the Error module) (@INC contains: /omd/sites/slave_bmw_ger/local/lib/perl5/lib/perl5 /omd/sites/slave_bmw_ger/lib/perl5/lib/perl5/x86_64-linux-gnu-thread-multi /omd/sites/slave_bmw_ger/lib/perl5/lib/perl5 /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.24.1 /usr/local/share/perl/5.24.1 /usr/lib/x86_64-linux-gnu/perl5/5.24 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.24 /usr/share/perl/5.24 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base) at lib/nagios/plugins/check_webinject line 34.
BEGIN failed--compilation aborted at lib/nagios/plugins/check_webinject line 34.

**Comment by gurubert** [comment]
Please report this as a bug against the checkmk Docker image.

**Comment by symentis-info** [comment]
Just for completeness: Error.pm bug is fixed in Docker version 1.6.0p5.cee
